schemas/webpagetest/webpagetest-run/webpagetest-run.1.parquetmr.txt

```diff
@@ -0,0 +1,19 @@
+message webpagetest-run {
+  required binary appName (UTF8);
+  required binary channel (UTF8);
+  required binary connection (UTF8);
+  required binary domain (UTF8);
```

Comment:
At the moment we're only testing landing pages, but I've some some suggestions that we may want to test content pages as they're more representative of what users are visiting. Perhaps a full URL option or path?

Reply:
Good point. We can just rename this to url, we're not handling millions of them so we should be fine without separate domain field?